src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -1609,8 +1609,8 @@ def aks_browse(cmd, client, resource_group_name, name, disable_browser=False,
 
     # verify the kube-dashboard addon was not disabled
     instance = client.get(resource_group_name, name)
-    addon_profiles = instance.addon_profiles or {}
-    addon_profile = addon_profiles.get("kubeDashboard", ManagedClusterAddonProfile(enabled=True))
+    addon_profiles = {k.lower(): v for k, v in (instance.addon_profiles or {}).items()}
+    addon_profile = addon_profiles.get("kubedashboard", ManagedClusterAddonProfile(enabled=True))
```

Comment:
shall we define it as a const? actually, azure-cli-extensions has extract the consts into _consts file. consider to adopt the same.
https://github.com/Azure/azure-cli-extensions/blob/master/src/aks-preview/azext_aks_preview/_consts.py

Reply:
sure